forte/base_classes/active_space_solver.h

```diff
@@ -130,6 +131,12 @@ class ActiveSpaceSolver {
     /// Set if read wave function from file as initial guess
     void set_read_initial_guess(bool read_guess) { read_initial_guess_ = read_guess; }
 
+    /// Return the coupling coefficients for a given state
+    CICouplingCoefficients coupling_coefficients(const StateInfo& state, int max_level = 2);
```

Comment:
I believe both these functions can be marked `const`.

Reply:
To do this, I have to use `at` as in the previous comment.